tox.ini

```diff
@@ -0,0 +1,52 @@
+[tox]
+envlist = py3, pycodestyle, pyflakes, pylint
+recreate = True
+
+[testenv]
+basepython = python3.6
+setenv =
+    FAKE_TRANSLATE = always
+deps =
+    urwid==1.2.1
+    PyYAML
+    attrs
+    systemd-python
+    jsonschema
+    pyudev
+    requests
+    requests-unixsocket
+    --editable=git+https://github.com/CanonicalLtd/probert@670c7258014eef13a971be0bf12206d96f2829f3#egg=probert
+
+
+[testenv:pycodestyle]
+deps =
+    pycodestyle==2.4.0
+commands = {envpython} -m pycodestyle \
+    {posargs:subiquity/ subiquitycore/ console_conf/}
+
+[testenv:pylint]
+deps =
+    {[testenv]deps}
+    pylint==1.9.1
+
+[testenv:py3]
+deps =
+    {[testenv]deps}
+    nose
+
+commands = {envpython} -m nose \
+    {posargs:subiquity/ subiquitycore/ console_conf/}
+
+
+[testenv:coverage]
+deps = {[testenv:py3]deps}
+    nose-timer
+    coverage
+commands = {envpython} -m nose \
+    --with-timer --timer-top-n 10 \
+    --with-coverage --cover-erase --cover-branches --cover-inclusive \
+    {posargs:subiquity/ subiquitycore/ console_conf/}
+
+[flake8]
+ignore=F403
+exclude = .tox,doc,*egg,.git,build
```

Comment:
I do not believe all these are actually necessary as it should default to: *.py and exclude defaults to:

 .svn,CVS,.bzr,.hg,.git,__pycache__,.tox"

per: http://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-exclude